docs/source/getting-started/installation.rst

```diff
@@ -75,6 +75,10 @@ If running :code:`conda install -c conda-forge google-cloud-sdk` produces the er
 
 Note: if you encounter *Authorization Error (Error 400: invalid_request)* with the url generated by :code:`gcloud auth login`, try installing the latest version of the `Google Cloud SDK <https://cloud.google.com/sdk/docs/install>`_ (e.g., with :code:`conda install -c conda-forge google-cloud-sdk`) on **your local machine** (which opened the browser) and rerun the command.
 
+.. note::
+
+  If you are using multiple GCP projects, you can set the project to use with :code:`gcloud config configurations activate <project_name>` (See `GCP docs <https://cloud.google.com/sdk/docs/configurations#activating_a_configuration>`_).
```

Comment:
Ah great catch. I updated a bit. Does this look better? I'm just worried that if it's in FAQ it may be hard to find for users.
<img width="835" alt="Screen Shot 2022-12-19 at 2 51 51 PM" src="https://user-images.githubusercontent.com/1206058/208542135-7a68f663-53f6-46f2-9764-002eea3717ab.png">